src/Client/App.fs

```diff
@@ -146,6 +148,7 @@ Program.mkProgram init update view
 #endif
 |> withReact "elmish-app"
 #if DEBUG
-|> Program.withDebugger
+// The debugger isn't support yet for Fable 2
```

Comment:
@MangelMaxime @alfonsogarciacaro what about this?

Reply:
@forki It's now working with Fable 2. 

You can update to latest version and use it like before.